sockets/1.2/modules/ROOT/pages/editorial/sockets-connector-studio.adoc

```diff
@@ -0,0 +1,271 @@
++
+[%header,cols=2]
+|===
+| Tab Name | Description
+| *General* | Set the *Host* and the *Port* required attributes, and configure any other optional fields.
+| *Buffer Configuration* | Optionally specify send and receive buffer size information.
+| *Connection* | Optionally select and configure one of the supported TCP protocols.
+| *Local Address Settings* | Optionally specify the local address.
+| *TLS* | Optionally specify the TLS information.
+| *Timeout Configuration* | Optionally specify the client and server timeout information.
+| *Advanced* | Optionally specify the reconnection information, including a reconnection strategy.
+|===
+
+[start=7]
+. Click *Test Connection* to confirm that Mule can connect with the specific server.
+. Click *OK*.
+
+Additionally, for the *Send and receive* operation, configure the MIME Type as follows:
+
+. Click the *Send and receive* operation from your flow.
+. On the *MIME Type* tab, select a MIME type from the drop-down menu field.
+
+In Studio, the Sockets Send operation and Send and receive operation in the flow look like this:
+
+.Sockets Send and Receive Operation
+image::socket-send-operation.png[Sockets Send and Receive operation]
+
+In Studio, configuring either the Send operation or the Send and receive operation, with a TCP connection to accept local incoming connections on port `8082` looks like this:
+
+.Sockets Send or Send and Receive Operation Configuration
+image::socket-sendreceive-configuration.png[Sockets Send or Send and Receive operation Configuration]
+
+In the *Configuration XML* tab, the Send operation configuration XML looks like this:
+
+[source,xml,linenums]
+----
+<sockets:request-config name="Sockets_Send_config">
+		<sockets:tcp-requester-connection host="localhost" port="8082" />
+	</sockets:request-config>
+
+	<flow name="SocketsFlow" >
+		<sockets:send config-ref="Sockets_Send_config"/>
+	</flow>
+----
+
+In Studio, configuring the MIME Type as `application/json` for the Send and receive operation looks like this:
+
+.Sockets MIME Type Configuration
+image::socket-mimetype-configuration.png[Sockets MIME type Configuration]
+
+In the *Configuration XML* tab, the Send and receive operation configuration XML looks like this:
+
+[source,xml,linenums]
+----
+<sockets:request-config name="Sockets_Send_config">
+	<sockets:tcp-requester-connection host="localhost" port="8082" />
+</sockets:request-config>
+
+<flow name="SocketsFlow" >
+	<sockets:send-and-receive config-ref="Sockets_Send_Receive_config" outputMimeType="application/json"/>
+</flow>
+----
+
+== Configure the Supported TCP Protocols
+
+Sockets Connector supports application-level protocols implemented on top of TCP. You must implement these protocols in both client and server ends so they can work properly. +
+By default, the Sockets Connector implements the Safe Protocol; however, you can configure a different protocol for the connector operations both in Anypoint Studio and XML.
```

Comment:
I looked up Safe Protocol to see if it is really a proper noun:

https://www.google.com/search?ei=22lOX9DxFpC60PEP3qWZ8Ak&q=what+is+the+safe+protocol+in+mulesoft&oq=what+is+the+safe+protocol+in+mulesoft&gs_lcp=CgZwc3ktYWIQAzoGCAAQFhAeOggIIRAWEB0QHjoFCCEQoAE6BwghEAoQoAFQ3itY2EBgvEJoAHAAeACAAXqIAdoLkgEDNS45mAEAoAEBqgEHZ3dzLXdpesABAQ&sclient=psy-ab&ved=0ahUKEwiQy_71o8jrAhUQHTQIHd5SBp4Q4dUDCA0&uact=5

It looks like there is ...

- element <safe-protocol>
- safe-protocol 
- Safe Protocol

Let me know? I'll stop marking it as a question from here to the end, so I don't "ditto" you to death!  ;-)